ci-operator/step-registry/ibmcloud/provision/vpc/endpoints-gateway/ibmcloud-provision-vpc-endpoints-gateway-ref.yaml

```diff
@@ -42,5 +45,25 @@ ref:
     default: ""
     documentation: |-
       GlobalTagging endpoint
+  - name: SERVICE_ENDPOINT_COSConfig
+    default: ""
+    documentation: |-
+      COSConfig endpoint
+  - name: SERVICE_ENDPOINT_GlobalCatalog
+    default: ""
+    documentation: |-
+      GlobalCatalog endpoint
+  - name: SERVICE_ENDPOINT_KeyProtect
+    default: ""
+    documentation: |-
+      KeyProtect endpoint
+  - name: SERVICE_ENDPOINT_CIS
```

Comment:
I did not see the default endpoint for CIS defined in `eps_default.json`, if we are not ready to test it, pls remove it.